_platforms/it/otello.md

```diff
@@ -2,6 +2,7 @@
 title: OTELLO 2.0
 subtitle: "Online Tax Refund at Exit: Light Lane Optimization"
 logo: /assets/images/logo-otello2@2x.png
+external_website: https://www.adm.gov.it/portale/dogane/operatore/ecustoms-aida/progetti-aida/otello
```

Comment:
This is a particularly long (and quite ugly) URL, resulting in this:
![Schermata da 2021-03-04 15-18-58](https://user-images.githubusercontent.com/25111463/109977401-107b0580-7cfd-11eb-8839-949e64549d2d.png)
which is suboptimal.
I think it could be nice to shrink it in some way, any ideas how? cc @bfabio since there are already discussions about something similar in https://github.com/italia/developers.italia.it/issues/775

Reply:
@libremente CSS ellipsize is the best solution